design/meta/process/process.md

```diff
@@ -0,0 +1,42 @@
+
+Every design has an editor. Often, the de facto editor is the person that has been leading discussions or championing a particular proposal. In cases where it is less obvious who the editor should be, someone may volunteer, or the group may nominate someone. The only outcome from this stage of the process is *agreement* on who the editor is.
+
+The editor forks the `rhombus-prototype` repo and creates a new document at the path `design/topic/filename.ext` -- using Markdown format or Scribble (rendered to HTML or Markdown). Initially, the design document is maintained *on the editor's fork* until it is ready to merge. That is, any edits to the design document must be made either by the editor or as pull requests against the editor's fork.
+
+Once a design document and its prototype have evolved to the point that they seem likely components of the Rhombus design, the pull request can be merged into the *Rhombus repo's main branch*. Discussion proceeds in this second phase as in the first. The switch from being a pull request (with recommended changes as part of the request) to a merged change (with recommended changes perhaps as new pull requests) simply reflects that different tools seem better suited to the earlier phase versus a later phase.
+
+An editor may at any time abdicate their responsibility to another project member, which can be decided by discussion in any convenient, even private, channel. But any change of editor *must be announced to the group* in a canonical channel.
+
+The Design Document
+-------------------
+
+The purpose of the document is to propose a particular way of doing things. In its discussion summary, a design document should showcase different ways of accomplishing the design's aims as have been discussed, and the summary should relate them to the consensus choices that are reflected in the design document. An editor is expected to both participate in dicussion and also synthesize and organize the record of discussion. Some alternative designs may be worth mentioning as part of a design's description, but as much as possible the disucssion summary as an appendix should be a complete and accurate reflection of the discussion.
+
+At the point where a design becomes wholly integrated into the Rhombus documentation, the original design document may become redundant except for the discussion summary. A redundant design description can be removed (replaced with a reference to documentation sections), but the discusssion summary should be preserved and maintained in perpetuity.
+
+Supporting Materials
+--------------------
+
+The pull request for an implementation may include changes to the Rhombus implementation and documentation. For additional code and material, the design may have a corresponding subfolder in the `design` folder in the Rhombus repo. For example, supporting material in the subfolder may include data gathered in the service of claims made in the design document, such as surveys, studies, or analyses.
```

Comment:
I would also add prototypes / proofs-of-concept here -- that is, standalone, runnable code that proves the design (e.g. the code in `equality/poc`), but need not be a modification of Rhombus itself. This ensures that provable designs need not wait to have a Rhombus implementation before they are convincing, and lowers the barrier to contributing usefully.